.github/workflows/jenkins-security-scan.yml

```diff
@@ -0,0 +1,22 @@
+name: Jenkins Security Scan
+
+on:
+  push:
+    branches:
+      - main
+      - master
+  pull_request:
+    types: [ opened, synchronize, reopened ]
+  workflow_dispatch:
+
+permissions:
+  security-events: write
+  contents: read
+  actions: read
+
+jobs:
+  security-scan:
+    uses: jenkins-infra/jenkins-security-scan/.github/workflows/jenkins-security-scan.yaml@v2
+    with:
+      java-cache: 'maven' # Optionally enable use of a build dependency cache. Specify 'maven' or 'gradle' as appropriate.
+      java-version: 17 # What version of Java to set up for the build.
```

Comment:
remove the default - [17 is used by default](https://github.com/jenkins-infra/jenkins-security-scan/blob/main/.github/workflows/jenkins-security-scan.yaml#L33)

Reply:
Yeah, this was changed after I created the PR. I have updated the config.